prospero-common/src/main/java/org/wildfly/prospero/wfchannel/MavenSessionManager.java

```diff
@@ -61,6 +61,11 @@ public MavenSessionManager(Optional<Path> provisioningRepo, boolean offline) thr
                 throw new ProvisioningException("Unable to create provisioning repository folder.", e);
             }
         }
+
+        if (System.getProperty("aether.offline.protocols") == null) {
+            System.setProperty("aether.offline.protocols", "file");
```

Comment:
> @spyrkob wdyt? Should we keep that UX improvement?

Yes, I'd like to leave it in. I wanted to do something similar and thought I'd need to filter list of repositories, so that's much neater :)